app/questionnaire/placeholder_transforms.py

```diff
@@ -56,9 +56,12 @@ def format_date(self, date_to_format: str, date_format: str) -> str:
         return formatted_datetime
 
     @staticmethod
-    def format_list(list_to_format: Sequence[str]) -> str:
-        formatted_list = "<ul>" + "".join(f"<li>{item}</li>" for item in list_to_format)
-        formatted_list += "</ul>"
+    def format_list(list_to_format: Sequence[str] | None) -> str:
+        formatted_list = (
+            f"<ul>{''.join(f'<li>{item}</li>' for item in list_to_format)}</ul>"
+            if list_to_format
+            else ""
+        )
```

Comment:
Would future-proof this if you wrap `list_to_format` in `self.remove_empty_from_list`

Reply:
Good shout - done